File: Aissgnment 27/Example4.c
/* 4.  Write a program which accept string from user and copy the  small character of that string into another
 string*/



#include<stdio.h>

void StrCpySmall(char *src,char *destination )
{
    while (*src !='\0')
    {
        if((*src>='a')&&(*src<='z'))
        {
          *destination=*src;
        destination++;  
        }
       
        src++;

    }
    *destination = '\0';   
}
 
 int main()
 {
 char Arr[30]="Marvellous Multi OS";
 char Brr[30];

 StrCpySmall(Arr,Brr);

    printf("%s",Brr);

    return 0;

 }

File: Assignment 53/Example4.java
// 4. write java program which accept two arrays from user and copy the contents of that array into another array
//and return new array.
      
import java. util.*;

class MyArray
{
    public int[] CopyArray(int Arr[])
    {
        int iSize = Arr.length;
       int[] copiedArr = new int[iSize];

      for (int i = 0; i < iSize; i++) 
      {
       copiedArr[i] = Arr[i];
     }

    return copiedArr;
    }
}

 class Example4
 {
     public static void main (String A [])
    {
       Scanner sobj=new Scanner(System.in);

        System.out.print("Enter size of first array: ");

        int size1 = sobj.nextInt();
        int Arr1[] = new int[size1];

        System.out.println("Enter elements of first array:");
        int i = 0;
        for (i = 0; i < size1; i++)
         {
            Arr1[i] = sobj.nextInt();
        }

        System.out.print("Enter size of second array: ");

        int size2 = sobj.nextInt();
        int Arr2[] = new int[size2];

        System.out.println("Enter elements of second array:\n");
         int j = 0;
        for (j = 0; j < size2; j++) 
        {
            Arr2[j] = sobj.nextInt();
        }

        MyArray obj = new MyArray();
          
      int[] copyArr1 = obj.CopyArray(Arr1);
        int[] copyArr2 = obj.CopyArray(Arr2);


        System.out.println("Copied first array:");
         
      for ( i = 0; i < copyArr1.length; i++) 
      {
         System.out.print(copyArr1[i] + " ");
     }
      System.out.println();

     System.out.println("Copied second array:");
     for ( i = 0; i < copyArr2.length; i++) 
    {
       System.out.print(copyArr2[i] + " ");
    }
    System.out.println();

     }
    
}